lib/13_xml_document.rb

```diff
@@ -1,3 +1,32 @@
 class XmlDocument
-  # TODO: your code goes here!
+  def initialize(indent = false)
+    @indent = indent
+    @indent_level = 0
+  end
+
+  def method_missing(method_name, options = {}, &block)
```

Comment:
Metaprogramming...the root of all evils...or at least massive rails refactors...